test/e2e/apimachinery/aggregator.go

```diff
@@ -87,19 +88,18 @@ var _ = SIGDescribe("Aggregator", func() {
 	})
 
 	/*
-		    Testname: aggregator-supports-the-sample-apiserver
-		    Description: Ensure that the sample-apiserver code from 1.10 and compiled against 1.10
-			will work on the current Aggregator/API-Server.
+	   Testname: aggregator-supports-the-sample-apiserver
+	   Description: Ensure that the sample-apiserver will work as an aggregated API server.
 	*/
-	framework.ConformanceIt("Should be able to support the 1.10 Sample API Server using the current Aggregator", func() {
-		// Testing a 1.10 version of the sample-apiserver
+	framework.ConformanceIt("Should be able to support the Sample API Server using the current Aggregator", func() {
+		// Testing a 1.17 version of the sample-apiserver
```

Comment:
If we aren't going to bump this every time, we should say `Should be able to support the 1.17 Sample API Server`.  If we are, we should say `Should be able to support the current Sample API Server`.  Both are valuable, we probably should have two tests.

Reply:
Added 1.17 back into the test name. The 1.17 level of the sample-apiserver is significant because it relies only on v1 APIs.

Testing the current level is trickier since it requires an image build.